examples/shader/dissolve.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build ignore
+
+package main
+
+var Time float
+var Cursor vec2
+var ImageSize vec2
```

Comment:
This was reverted to `ScreenImage`. Sorry!

Reply:
since this is not used, this is only for consistency?

Reply:
This is only for consistency.